src/delimiter.js

```diff
@@ -115,12 +155,16 @@ const makeLargeDelim = function(delim, size, center, options, mode, classes) {
  * Make an inner span with the given offset and in the given font. This is used
  * in `makeStackedDelim` to make the stacking pieces for the delimiter.
  */
-const makeInner = function(symbol, font, mode) {
+const makeInner = function(
+    symbol: string,
+    font: "Size1-Regular" | "Size4-Regular",
```

Comment:
I'm surprised we only call this with `Size1` or `Size4` since there are sizes in between.